### PR TITLE
Re-implemented a mostly working menu

### PR DESCRIPTION
**Pull request by sirmackan**

### CODE REVIEW

File: app/app.component.html [hunk 1]
```diff
@@ -20,6 +20,90 @@
...
+              <div class="user-info-image" [ngStyle]="{'background-image': 'url(' + ((user.images && user.images[0] && user.images[0].imageUrl) || 'assets/images/placeholder-profile-image.png') + ')'}"></div>
+              {{user.firstName}} {{user.lastName}}
+            </div>
+            <div *ngIf="!user">
+              <a routerLink="/user/register" class="create-user medium-button-primary-light">Create User</a>
+            </div>
+          </div>
+        </div>
+
+        <div *ngIf="isLanguageMenuVisible" class="language-menu-container" style="overflow-y: scroll; text-align: center;">
```
Comment (buren) [inline app/app.component.html]: Inline style? :)

File: app/app.component.html [hunk 2]
```diff
@@ -20,6 +20,90 @@
...
+          <div style="text-align: right;"><i class="fa fa-times fa-2x" (click)="isLanguageMenuVisible = false"></i></div>
+          <div class="just-arrived-logo-container" style="margin-bottom: 2em;">
+            <img class="language-logo-img" src="assets/images/logo.png" alt="Just Arrived" style="height: 50px;" />
```
Comment (buren) [inline app/app.component.html]: Inline style? :)

File: app/app.component.html [hunk 3]
```diff
@@ -20,6 +20,90 @@
...
+
+          </div>
+          <div class="language-information">
+            <p>{{'menu.language.text_1' | translate}}</p>
+            <p>{{'menu.language.text_2' | translate}}</p>
+          </div>
+          <div class="credentials">
+            <p class="prominent">{{'menu.language.translation_by_probono' | translate}}:</p>
+            <img src="assets/images/menu_semantix_logo.png" alt="Semantix"/>
+            <img src="assets/images/menu_skrivasbyra_logo.png" alt="Stockholms.Skrivbyrå"/>
```
Comment (buren) [inline app/app.component.html]: Remove this line :) ☝️

File: app/app.component.html [hunk 4]
```diff
@@ -20,6 +20,90 @@
...
+            <p class="start_name">Farshid Fadaee</p>
+            <p>Nick Chipperfield</p>
+            <p>Hans-Erik Täpp</p>
```
Comment (buren) [inline app/app.component.html]: Hardcoded values? :) ☝️

**Comment by tikkat** [comment]
👍

**Pull request merged**